app/uk/gov/hmrc/exports/models/declaration/Locations.scala

```diff
@@ -30,12 +30,18 @@ object GoodsLocation {
   implicit val format: OFormat[GoodsLocation] = Json.format[GoodsLocation]
 }
 
-case class OfficeOfExit(officeId: Option[String], presentationOfficeId: Option[String], circumstancesCode: Option[String])
+case class OfficeOfExit(officeId: Option[String], presentationOfficeId: Option[String], circumstancesCode: Option[String], answer: String)
 
 object OfficeOfExit {
   implicit val format: OFormat[OfficeOfExit] = Json.format[OfficeOfExit]
 }
 
+case class OfficeOfExitOutsideUk(officeId: String)
```

Comment:
If you _are_ going to use a new model, consider making the officeId and Option[String] - otherwise we'll never be able to make your new outside Uk office screen optional.

Reply:
This has been changed as per suggestion above.